### PR TITLE
Reorganize simulcast functionality

### PR DESCRIPTION
**Pull request by aboba**
This is an attempt to reorganize the simulcast-related material so that it is easier to add more detail in future.

Includes fix for Issue https://github.com/w3c/webrtc-pc/issues/1987

Changes can be viewed here: https://rawgit.com/aboba/webrtc-pc/master/webrtc.html

***
<a href="https://pr-preview.s3.amazonaws.com/aboba/webrtc-pc/pull/1990.html" title="Last updated on Sep 20, 2018, 5:43 AM GMT (28c0c7f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/1990/35328b6...aboba:28c0c7f.html" title="Last updated on Sep 20, 2018, 5:43 AM GMT (28c0c7f)">Diff</a>

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -7338,6 +7355,81 @@ <h2>Methods</h2>
...
+  scaleResolutionDownBy: 1.0
+}, {
+  // Simulcast layer at one half scale
+  rid: "h",
+  scaleResolutionDownBy: 2.0
+}, {
+  // Simulcast layer at one quarter scale
+  rid: "q",
+  scaleResolutionDownBy: 4.0
+}];
```
Comment (jan-ivar) [inline webrtc.html]: This example looks identical to [10.4 Simulcast Example](http://w3c.github.io/webrtc-pc/#simulcast-example). Do we need both?

Would it suffice to decorate the existing example with these comments?

File: webrtc.html [hunk 2]
```diff
@@ -7338,6 +7355,81 @@ <h2>Methods</h2>
...
+  active: false
+}, {
+  rid: "h",
+  scaleResolutionDownBy: 2.0,
+  active: false
+}, {
+  rid: "q",
+  scaleResolutionDownBy: 4.0,
+  active: true
+}];
```
Comment (jan-ivar) [inline webrtc.html]: Should we maybe use the same formatting as in [10.4 Simulcast Example](http://w3c.github.io/webrtc-pc/#simulcast-example)? E.g.:
```
    // Example of 3-layer spatial simulcast with all but the lowest resolution layer disabled
    pc.addTransceiver('video', {
      direction: 'sendonly',
      sendEncodings: [
        {rid: 'f', active: false},
        {rid: 'h', active: false, scaleResolutionDownBy: 2.0},
        {rid: 'q', active, true, scaleResolutionDownBy: 4.0}
      ]
    });
```
At minimum we should use single-quotes like the other examples.

File: webrtc.html [hunk 3]
```diff
@@ -7338,6 +7355,81 @@ <h2>Methods</h2>
           </dl>
         </section>
       </div>
+      <section>
+        <h3>Simulcast functionality</h3>
+    <p>Simulcast functionality is provided via the
+    <code>addTransceiver</code> method of the <code><a>RTCPeerConnection</a></code> object
+    and the <code>setParameters</code> method of the <code><a>RTCRtpSender</a></code> object.</p>
+    <p>The <code>addTransceiver</code> method establishes the "simulcast envelope" which includes
+    the maximum number of simulcast streams that can be sent, as well as the ordering of the <code>encodings</code>.
```
[the remark this answers is not in the document]
Comment (aboba) [reply webrtc.html]: I added a parenthetical definition and link to it.  We will need further discussion on exactly what can and cannot be changed (some open issues refer to this)

File: webrtc.html (hunk 1 above)
Comment (aboba) [reply webrtc.html]: Deleted the overlapping example and added a different one.

File: webrtc.html (hunk 2 above)
Comment (aboba) [reply webrtc.html]: Fixed the format.

File: webrtc.html [hunk 4]
```diff
@@ -7338,6 +7355,81 @@ <h2>Methods</h2>
...
+    encodings and the parameters retrieved via the transceiver's
+    <code>receiver.getParameters()</code> will reflect the
+    encodings negotiated.</p>
+    <p class="note">An <code><a>RTCRtpReceiver</a></code> can receive multiple RTP streams in a scenario
+    where a Selective Forwarding Unit (SFU) switches between simulcast streams it receives from user agents.
+    If the SFU does not rewrite RTP headers so as to arrange the switched streams into a single RTP
+    stream prior to forwarding, the <code><a>RTCRtpReceiver</a></code> will receive packets from
+    distinct RTP streams, each with their own SSRC and sequence number space.
+    While the SFU may only forward a single RTP stream at any given time, due to reordering packets
+    from multiple RTP streams can become intermingled at the receiver.
```
[the remark this answers is not in the document]
Comment (aboba) [reply webrtc.html]: Fixed.

**Pull request merged**